README.md

```diff
@@ -96,5 +96,13 @@ You can do this portion of the exercise in any tool you'd like. Excel, Google Sh
 
 ## Exploratory Problem
 
+- Here is a bonus challenge which will definitely attract our attention if you answer.  The challenge is to analyze the bookings data and provide recommendations to our sales process to improve the Close Rate.  
 
+- Definition: Close Rate = Number of opportunities closed won / Number of opportunities
 
+
+### instructions
+
+- Here’s a last 12 month denormalized data set on our Bookings [CSV file](/bonus_challenge/data/Bookings_Data.csv).
+
+- Submit your recommendations along with the analysis in any Word or PowerPoint document in the [recommendations directory](/bonus_challenge/recommendations).
```

Comment:
Recommendations should be a brief `Word` or `PowerPoint` document - analysis can be in excel, sql, or notebook etc.